code/datums/uplink_item.dm

```diff
@@ -684,7 +689,7 @@ var/list/uplink_items = list()
 	desc = "A crate containing 50 telecrystals worth of random syndicate leftovers."
 	cost = 20
 	item = /obj/item/weapon/storage/box/syndicate
-	excludefrom = list(/datum/game_mode/nuclear)
+	excludefrom = list(/datum/game_mode/nuclear,/datum/game_mode/gang)

```

Comment:
of note is that because of the way uplinks work, the surplus crate actually only dispenses items from the current mode.